blogService/src/main/java/com/blog/blogservice/service/impl/ArticleServiceImpl.java

```diff
@@ -43,8 +51,14 @@ private boolean isUserAbleToCreatePost(final ArticleDto articleDto, final Long b
     @Override
     public ArticleDto getArticleById(final Long articleId) {
         Optional<Article> articleOptional = articleRepository.findById(articleId);
-        return articleOptional.map(article -> articleMapper.map(article, ArticleDto.class))
-                .orElse(null);
+        if (articleOptional.isPresent()) {
+            List<CommentDto> commentDtos = articleOptional.get().getComments().stream()
+                    .map(comment -> commentMapper.map(comment, CommentDto.class))
+                    .collect(Collectors.toList());
+            articleOptional.map(article -> articleMapper.map(article, ArticleDto.class));
+           // brakuje return rezultatu,ale utknęłam
```

Comment:
tutaj wstaw 
  articleDto.setComments(comments);
return articleDto;

przecież nawet napisałam ci to w poprzednim komantarzu, nie przeczytałas lub niedokładnie, ajajajaj :p